utilities/sanity.py

```
@@ -315,6 +316,9 @@ def cluster_sanity(
             hco_namespace=hco_namespace,
         )
 
+        if taint_conditions := is_hco_tainted(admin_client=admin_client, hco_namespace=hco_namespace.name):
+            raise ClusterSanityError(err_str=f"HCO has TaintedConfiguration: {taint_conditions}.")
```

Ruff (0.16.1)

[warning] Abstract `raise` to an inner function

(TRY301)